### ISSUE TITLE
Include system.img in /backup

### ISSUE THREAD

**Issue by fonji** [cookiengineer/alcatel-6015x#3]
Hello,

I completely bricked my phone to the point where the official mobile upgrade utility doesn't detect it. I found a way to get it to fastboot, and I also can access the original recovery tool.

But I couldn't find a working system.img file anywhere. Does anyone has a backup or could create an image? I think that it is missing in /backup.
Do you think that compiling from the alcatel sources could get me out of this?

Anyway, thank you for the help you provide to all of us who are lost.

NB : of course I didn't backup my phone prior to bricking it because the tutorial I followed didn't have this step and I thought that I always could use the recovery tool... Silly me.

**Comment by llacroix** [comment]
Did you turn off the phone completely prior to run the alcatel tool?

**Comment by fonji** [comment]
Yes I did, but as soon as I plug the USB cable in, the phone boots and reboots and rereboots etc.
So the tool never detects it.
Thank you!
